zebra/zebra_vxlan.c

```diff
@@ -269,7 +269,7 @@ static void zevpn_print_neigh_hash_all_evpn(struct hash_bucket *bucket,
 	wctx.zevpn = zevpn;
 	wctx.vty = vty;
 	wctx.addr_width = 15;
-	wctx.r_vtep_width = 39;
+	wctx.r_vtep_width = INET6_GUA_ADDRSTRLEN;
```

Comment:
so ... don't get it: why replace "39" but leave "15" in-place? this is sort of the point about what the lib macros are for: they're not really for local micro vty output formatting?

Reply:
INET6_ADDRSTRLEN is used for GUA and LL width already defined macro, there isn't one for printing GUA address which max can be 40 characters. It is a useful to use going forward wherever possible, and not use the higher length of 46 which contains LL  + sight origin. 
In my opinion it is useful to define new macro and utilize it.